tests/test_integrationToDoApp.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+from dotenv import find_dotenv,load_dotenv
+from todo_app.app import create_app
+from unittest.mock import patch, Mock
+
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    test_app = create_app()
+    with test_app.test_client() as client:
+        yield client
+
+
+@patch('requests.request')
+def test_index_page(mock_get_requests, client):
+    mock_get_requests.side_effect = mockResponse
+
+    response = client.get('/')
+    responseCode = response.status_code
+    responseHtml = response.data.decode()
+    assert responseCode == 200
+    assert "testDoingTask" in responseHtml
+    assert "testDoneTask" in responseHtml
+    assert "testDoingTask_UID:1548993486125" in responseHtml
+
+
+
+
+
+def mockResponse(method, url, params):
+
+        status=200,
+        content="CONTENT",
+        
+        raise_for_status=None
+
+        mock_resp = Mock()
+        # mock raise_for_status call w/optional error
+        mock_resp.raise_for_status = Mock()
+        if raise_for_status:
+            mock_resp.raise_for_status.side_effect = raise_for_status
```

Comment:
This will never be triggered since `raise_for_status` is always None - this is currently unused so I'd  be inclined to get rid of it